rllib/algorithms/tests/test_algorithm_export_checkpoint.py

```diff
@@ -95,9 +90,6 @@ def tearDownClass(cls) -> None:
     def test_save_appo_multi_agent(self):
         save_test("APPO", "torch", multi_agent=True)
 
-    def test_save_ppo(self):
```

Comment:
new API stack: Already covered extensively (including extracting RLModule checkpoint and testing it) in `checkpoints/` subfolder of the example scripts.